forc-pkg/src/pkg.rs

```diff
@@ -10,7 +10,7 @@ use forc_util::{
 };
 // Using `fuel_tx` directly instead of `fuel_gql_client` transitively is causing some weird issue.
 // See https://github.com/FuelLabs/sway/issues/2659
```

Comment:
```suggestion
// Using `fuel_tx` directly instead of `fuel_gql_client` transitively is causing some weird issue.
// See https://github.com/FuelLabs/sway/issues/2659
```

Maybe we can also remove this as this is solving the mentioned issue.

Reply:
Good point! Removed.